COTL_API/CustomStructures/StructurePatches.cs

```diff
@@ -216,5 +220,25 @@ public static bool StructuresData_CanBeFlipped(StructureBrain.TYPES type, ref bo
         if (!CustomStructures.ContainsKey(type)) return true;
         __result = CustomStructures[type].CanBeFlipped();
         return false;
+    }
+
+    [HarmonyPatch(typeof(Structure), nameof(Structure.Start))]
+    [HarmonyPostfix]
+    public static void Structure_Start(Structure __instance)
+    {
+
+        if (!CustomStructures.ContainsKey(__instance.Type)) return;
+
+        
+        if (CustomStructures[__instance.Type].Interaction != null)
+        {
+            Plugin.Logger.LogDebug("adding structure interaction " + CustomStructures[__instance.Type].Interaction);
+            var parent = __instance.GetComponentInParent<Transform>();
+            parent.gameObject.AddComponent(CustomStructures[__instance.Type].Interaction);
+        }
+
```

Comment:
What's with the extra space?

Reply:
oops, removed some stuff without removing the spaces, fixed that.